### PR TITLE
Updated GDScript grammar from setget to set/get

### PR DESCRIPTION
**Pull request by solorvox**
Fixes for set/get from older setget for GDScript grammar and tutorials. 
Should fix issue: https://github.com/godotengine/godot-docs/issues/6265

### CODE REVIEW

File: contributing/development/file_formats/gdscript_grammar.rst [hunk 1]
```diff
@@ -37,10 +37,14 @@ for reference purposes.
         | "tool"
         ;
 
-    classVarDecl = [ "onready" ] [ export ] "var" IDENTIFIER [ ":" typeHint ]
-        [ "=" expression ] [ setget ] NEWLINE ;
-    setget = "setget" [ IDENTIFIER ] [ "," IDENTIFIER] ;
-    export = "export" [ "(" [ BUILTINTYPE | IDENTIFIER { "," literal } ] ")" ] ;
+    classVarDecl = [ @onready ] [ @export ] "var" IDENTIFIER [ ":" typeHint ]
+    [ "=" expression ] IDENTIFIER
```
Comment (jmdejong) [inline contributing/development/file_formats/gdscript_grammar.rst:41]: I think that `IDENTIFIER` should not be here. I'm still learning the syntax myself, but I don't see it anywhere in your examples

Comment (solorvox) [reply contributing/development/file_formats/gdscript_grammar.rst:41]: Do you mean the first or second occurrence?  I was trying to follow the EBNF wiki.  But also using the previous setget syntax from the manual page.  What I thought it `IDENTIFIER` meant was for example `var foo = bar`.  Where `foo` was the first `IDENTIFIER` and `bar` was the second being an identifier to another variable.  But `bar` could of course be replaced with an expression as well.  Happy to take suggestions as EBNF wasn't completely clear to me either. :)

Comment (jmdejong) [reply contributing/development/file_formats/gdscript_grammar.rst:41]: I think in your example `bar` is part of the expression that is already defined by `[ "=" expression ]`.

The current definition would match `var foo bar` or `var foo = 123 bar` or `@export var foo  : int = 123 bar` (with `bar` as the second identifier).

**Comment by solorvox** [comment]
How does this commit look?

**Comment by mhilbrunner** [comment]
The remaining changes here were merged as part of https://github.com/godotengine/godot-docs/pull/9546. Thanks for contributing!

**Pull request closed without merging**